packages/components/src/components/composite-checkbox/composite-checkbox.scss

```diff
@@ -0,0 +1,15 @@
+.composite-checkbox {
+    border-radius: 0.4rem;
+    border: 1px solid var(--general-hover);
+    padding: 1.6rem;
+    cursor: pointer;
+
+    &__description {
+        @include typeface(--xsmall-left-normal-black);
+        color: var(--text-general);
```

Comment:
Aren't we going to end up with duplicated color properties? 
Albeit, since `var(--text-general)` it wins the conflict.

Reply:
I don't think so, line 9 will override what ever is that typeface is returning.

Reply:
Per discussion, will mark this as resolved. Will revisit this later to remove `color` param from the mixin. As it creates a weird duplicate look.